openshift-ironic.spec

```diff
@@ -0,0 +1,293 @@
+%global pyver 3
+%global full_release ironic-%{version}
+
+%{!?upstream_version: %global upstream_version %{version}%{?milestone}}
+
+Name:           openshift-ironic
+Epoch:          1
+Summary:        OpenShift Baremetal Hypervisor API (ironic)
+# (rpittau) we probably need to change this
```

Comment:
What I understood is that we can have a fake value here (e.g. DLRN uses XXX)

Reply:
you're right, the version is overridden by OCP building system, I will just put XXX